UploadAndAnnotateImages/force-app/main/default/lwc/darkInput/darkInput.js

```diff
@@ -0,0 +1,93 @@
+import { api, LightningElement } from "lwc";
+
+export default class DarkInput extends LightningElement {
+  @api
+  label;
+
+  @api
+  placeholder;
+
+  @api
+  required;
+
+  @api
+  multiline;
+
+  // You must specify "multiline=true" when changing the number of rows
+  @api
+  rows = 1;
```

Comment:
Yes I also thought about it when I wrote this code, but eventually I decided to keep it like this because if I add these setters the end result of the following code is unclear:
```
<c-dark-input multiline=false rows=3></c-dark-input>
```
Which setter will win? The `multiline` setter will set `rows=1`? Or the `rows` setter will set `multiline=true`?
So I decided that it's better to be explicit here.